cpp/src/parquet/arrow/test_util.h

```diff
@@ -65,12 +66,30 @@ struct Decimal256WithPrecisionAndScale {
   static constexpr int32_t scale = PRECISION - 1;
 };
 
+inline void RandomHalfFloatValues(int64_t n, uint32_t seed,
```

Comment:
We don't need to line the definition here, can we put in a `.cc` file instead?

Reply:
We could either create a corresponding `parquet/arrow/test_util.cc` or put it in `parquet/test_util.cc`. I'm not sure if the former is worth doing in this case, though.

Reply:
Given this is not Arrow-specific, putting it in `parquet/test_util.cc` would be fine (should probably also move the declaration then).